src/iOS.Core/Renderers/CollectionView/ExtendedGroupableItemsViewController.cs

```diff
@@ -38,10 +38,11 @@ protected override void UpdateTemplatedCell(TemplatedCell cell, NSIndexPath inde
                     // Do nothing in here, this is temporary to get more info about the crash, if the logger fails, we want to get the info
                     // by crashing with the original exception and not the logger one
                 }
-                if (ex is IndexOutOfRangeException)
+                if (ex is IndexOutOfRangeException || ex is ArgumentOutOfRangeException)
```

Comment:
ℹ️ Bare in mind that this will apply only to `CollectionView`s that have `ExtraDataForLogging` set, if this is fine then approving.